<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModelBrowserWidget</class>
 <widget class="QWidget" name="ModelBrowserWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>888</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>模型:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="mModels">
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="insertPolicy">
        <enum>QComboBox::NoInsert</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>设置:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="mSets"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mLoadFromFile">
       <property name="text">
        <string>从文件加载</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QWidget" name="widget_3" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="mViewWidget" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>1</horstretch>
              <verstretch>1</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="mGridBox">
            <property name="title">
             <string>Grid</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <layout class="QFormLayout" name="formLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>对比</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>X-偏移</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QDoubleSpinBox" name="mX_Offset">
               <property name="minimum">
                <double>-100.000000000000000</double>
               </property>
               <property name="maximum">
                <double>100.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="mGridAlignment"/>
             </item>
             <item row="2" column="1">
              <widget class="QDoubleSpinBox" name="mY_Offset">
               <property name="minimum">
                <double>-100.000000000000000</double>
               </property>
               <property name="maximum">
                <double>100.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QDoubleSpinBox" name="mZ_Offset">
               <property name="minimum">
                <double>-100.000000000000000</double>
               </property>
               <property name="maximum">
                <double>100.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Y-偏移</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Z-偏移</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="mDefinitionTab">
          <attribute name="title">
           <string>定义</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QTextEdit" name="mDefinition"/>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="mPreview">
               <property name="text">
                <string>预览</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="mApply">
               <property name="text">
                <string>应用</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="mModelGraphTab">
          <attribute name="title">
           <string>模型图</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QTreeView" name="mModelGraph">
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>mModels</tabstop>
  <tabstop>mSets</tabstop>
  <tabstop>mGridBox</tabstop>
  <tabstop>mGridAlignment</tabstop>
  <tabstop>mX_Offset</tabstop>
  <tabstop>mY_Offset</tabstop>
  <tabstop>mZ_Offset</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>mDefinition</tabstop>
  <tabstop>mPreview</tabstop>
  <tabstop>mApply</tabstop>
  <tabstop>mLoadFromFile</tabstop>
  <tabstop>mModelGraph</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
